apps/crn-server/test/fixtures/teams.fixtures.ts

```diff
@@ -79,7 +81,9 @@ export const getContentfulGraphqlTeam = (): NonNullable<
   inactiveSince: null,
   projectTitle:
     'The genome-microbiome axis in the cause of Parkinson disease: Mechanistic insights and therapeutic implications from experimental models and a genetically stratified patient population.',
-  expertiseAndResourceTags: ['Animal resources'],
+  researchTagsCollection: {
+    items: [],
```

Comment:
Maybe we could leave a tag here?
```suggestion
    items: [
    { sys: { id: 'tag-1' }, name: 'Animal resources 1' },
    ],
```